cv32e40s/tests/programs/custom/mhpmcounter_write_test/mhpmcounter_write_test.c

```diff
@@ -0,0 +1,241 @@
+// Copyright 2023 Silicon Labs, Inc.
+//
+// SPDX-License-Identifier: Apache-2.0 WITH SHL-2.1
+//
+// Licensed under the Solderpad Hardware License v 2.1 (the "License"); you may
+// not use this file except in compliance with the License, or, at your option,
+// the Apache License version 2.0.
+//
+// You may obtain a copy of the License at
+// https://solderpad.org/licenses/SHL-2.1/
+//
+// Unless required by applicable law or agreed to in writing, any work
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <stdio.h>
+#include <stdint.h>
+#include <stdlib.h>
+
+int write_mhpmcounters()
+{
+    uint32_t reg = 0;
+    __asm__ volatile("mv %0, x0" : "=r"(reg));
+    __asm__ volatile("not %0, %0" :: "r"(reg));
+    __asm__ volatile("csrrs x0, 0xB03, %0" :: "r"(reg));
```

Comment:
All the addresses are "magic numbers". It would be better if these were defines, and I think they are defined already? 

I don't think this is blocking, and can be ignored if we are in to much of a hurry.

Reply:
Not sure if "defines" are needed.
The assembly code usually recognizes names like `mseccfg`, `mtvt`, etc.

Reply:
FIXED